src/installation/liveimages/guide-chroot.md

```diff
@@ -0,0 +1,388 @@
+know what this means, it's recommended to choose ext4.
+
+```
+# mkfs.ext4 /dev/sd3
+# mkfs.ext4 /dev/sd4
+```
+
+### Mount the file systems
+
+First, mount the root partition to `/mnt`:
+
+```
+# mount /dev/sda3 /mnt
+```
+
+Then, create directories at the root of the partition for each mountpoint.
+
+```
+# mkdir --parents /mnt/boot/efi
+# mkdir /mnt/home
+```
+
+Mount the remaining partitions, accordingly.
+
+```
+# mount /dev/sda2 /mnt/boot/efi
+# mount /dev/sda4 /mnt/home
+```
+
+Verify all the partitions have been mounted correctly with
+[`lsblk(8)`](https://man.voidlinux.org/lsblk.8).
+
+```
+$ lsblk
+NAME   MAJ:MIN RM    SIZE RO TYPE MOUNTPOINT
+sda      8:0    1  238.5G  0 disk
+├─sda1   8:1    1      8G  0 part [SWAP]
+├─sda2   8:1    1    500M  0 part /mnt/boot/efi
+├─sda3   8:1    1     20G  0 part /mnt
+└─sda4   8:1    1     40G  0 part /mnt/home
+```
+
+## Installation
+
+Now, it's time to install the base system! If you'd like to perform an
+installation of an architecture different to the one you are running on right
+now, you must export the `XBPS_ARCH` environment variable accordingy, for
+example if you booted via *glibc* media and want to install a *musl-libc* system
+or vice-versa.
+
+To install a *musl-libc* system:
+
+```
+export XBPS_ARCH=x86_64-musl
+```
+
+To install a *glibc* system:
+
+```
+export XBPS_ARCH=x86_64
+```
+
+Then, install the `base-system` package group as well as `grub-x86_64-efi` for
+UEFI systems or `grub` otherwise. You may also choose not to install a
+bootloader at this time.
+
+```
+# xbps-install -S -R https://repo.voidlinux.eu/current -r /mnt base-system grub-x86_64-efi
+```
+
+The flags in the above command have the following meanings, from
+[xbps-install(1)](https://man.voidlinux.org/xbps-install):
+
+- `-S`: Synchronize remote repository index files.
+- `-R`: Enable repository mode. This mode explicitly looks in repositories,ther than looking in the target root directory.
+- `-r <dir>`: Specifies a full path for the target root directory.
+
+## Chroot
+
+Now, mount the remaining necessary filesystems so that `/proc`, `/dev`, and
+`/sys` are available after changing the apparent root.
+
+```
+# mount -t proc proc /mnt/proc
+# mount -t sysfs sys /mnt/sys
+# mount -o bind /dev /mnt/dev
+# mount -t devpts pts /mnt/dev/pts
+```
+
+Copy the `/etc/resolv.conf` file to the new system, so that DNS addresses can be
+resolved.
+
+```
+# cp -L /etc/resolv.conf /mnt/etc/
+```
+
+Finally, change root into the new system, and source the new shell profile. You
+can also change the prompt temporarily to remind yourself  while you're inside
+the chroot jail.
+
+```
+# chroot /mnt /bin/bash
+# . /etc/profile
+# export PS1="(chroot) \w \\$ "
+```
+
+## Configure the system
+
+### Root user
+
+Create a password for the root user, and set the correct permissions.
+
+```
+# passwd root
+# chown root:root /
+# chmod 755 /
+```
+
+### Fstab
```

Comment:
maybe make a note about optionally adding a tmpfs?

Reply:
Sounds good. Btw, the tmpfs entry is already in the `fstab` I should differentiate it somehow from the entries I added

Reply:
If it is already there when you do a chroot installation, then you don't have to mention it I guess.